docs/src/man/hosting.md

```diff
@@ -192,7 +192,7 @@ jobs:
   build:
     permissions:
       contents: write
-      statuses: write
```

Comment:
Is this implied by `contents: write` then?

Reply:
I'll re-add this as it's still needed. However, if you don't have this permission you just get a silent failure:

> Without: statuses: write the ["Create a commit status"](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status) API call [used by Documenter.jl](https://github.com/JuliaDocs/Documenter.jl/blob/346c622bf7eb7bc70ab7de6481650285de599921/src/deployconfig.jl#L504C26-L504C54) will [silently fail](https://github.com/JuliaDocs/Documenter.jl/blob/346c622bf7eb7bc70ab7de6481650285de599921/src/deployconfig.jl#L444-L466).